tools/ClassGen/NodeGen.cpp

```diff
@@ -54,6 +54,31 @@ int main(int argc, char **argv) {
                     "this node and all of its ancestor nodes. Generally "
                     "intended to save the final result of a network.");
 
+  BB.newNode("Send")
+      .addInput("Input")
+      .addInput("Address")
```

Comment:
Just an idea: May be address should be a string? It could encode e.g. a transport and an address, e.g. `"memory://buffer_address"`, `"tcp://1.2.3.4:8080"`, `"PCIe://..."`, etc. This way you could specify which kind of channel to use for the communication.